Makefile

```diff
@@ -10,7 +10,8 @@ build:
 	cd $(EXTENSION_DIR); \
 	gnome-extensions pack --force --podir=locale \
 	                      --extra-source=preferences/ \
-	                      --extra-source=icons/; \
+	                      --extra-source=icons/ \
+												--extra-source=mprisMediaPlayer2.js; \
```

Comment:
Is there a reason this is so heavily indented?

Reply:
It's not in my ide. I see what happend. My line is all tabs. Other lines are tab + a bunch of spaces. Will fix.